Add Adyen extension documentation (#825)
front-commerce/developers.front-commerce.com: docs/03-extensions/adyen/reference/environment-variables.mdx

---
title: "Environment variables"
description:
  "This reference lists environment variables used by the Adyen extension"
sidebar_position: 1
---

<p>{frontMatter.description}</p>

## Debug

> This extension will output some debug informations if Front-Commerce is run with

Reviewer: ```suggestion
This extension will output some debug information when Front-Commerce is ran with
```

the `DEBUG` environment variable containing `front-commerce:payment:adyen`.

> ## Required environment variables

Reviewer: **issue:** these variables aren't required every time. Only in some flavors.

**suggestion:** split this section with variables per flavor
